src/data_collectors.js

```diff
@@ -17,14 +43,90 @@ function collectDialectData (doc, ctx, acc) {
     dialectData.slug = utils.slugify(dialectData.name)
     dialectData.htmlName = `${dialectData.slug}.html`
     console.log(`Collecting nodes info for dialect ${id}`)
-    dialectData.nodeMappings = collectNodesData(doc, dialectData, ctx)
+    dialectData.nodeMappings = collectNodesData(
+      doc, dialectData, ctx, ontologyTerms)
       .sort(utils.nameSorter)
     return dialectData
   }
 }
 
 /* Collects dialect nodeMappings data. */
-function collectNodesData (doc, dialectData, ctx) {
+function collectVocabularyNodesData (doc, dialectData, ctx) {
```

Comment:
Please update a docstring and comment above each `doc.queryAll` what that block collects data for.

Reply:
Added!